week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,7 +1,33 @@
 // Implement the missing functionality according to the task description.
 // Hint: Remember which function you need to call when it's time to alarm.
 
-function setAlarm() { }
+
+function setAlarm() {
+  let time = document.getElementById("alarmSet").value
+  let timeMin = Math.floor((time / 60))
+  let timeSecond = (time % 60)
+  alarmInt = setInterval(() => {
+    if (timeSecond === 0 && timeMin > 0) {
+      timeMin--;
+      timeSecond = 59;
+    } else if (timeSecond > 0) {
+      timeSecond--;
```

Comment:
Good work, will try to do more separation, like a function for time converting maybe 
trying to apply the S in SOLID principles